nestkernel/node_manager.cpp

```diff
@@ -724,15 +720,13 @@ void
 NodeManager::finalize_nodes()
 {
 #ifdef _OPENMP
-  LOG( M_INFO, "NodeManager::finalize_nodes()", " using OpenMP." );
-// parallel section begins
 #pragma omp parallel
   {
     index t = kernel().vp_manager.get_thread_id();
-#else
-    for ( index t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
-    {
-#endif
+#else // clang-format off
+  for ( index t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
+  {
```

Comment:
No, it shouldn't. Depending on the definition of `_OPENMP`, the compiler either sees
```
#pragma omp parallel
  {
```
or
```
  for (...)
  {
```
The `for` statement in line in (new) line 727 is _not_ inside the `#pragma omp parallel` block, but replaces it conditionally. So the indentation level of the opening curly brace in line 724 and (new) line 728 have to match.

As `clang-format` does apparently not take conditional compilation into account, I had to disable it to get the correct indentation.